pkg/action/executor/rescheduler.go

```diff
@@ -25,139 +24,97 @@ func NewReScheduler(ae TurboK8sActionExecutor, sccAllowedSet map[string]struct{}
 	}
 }
 
-//Note: the error info will be shown in UI
+// Execute executes the move action. The error message will be shown in UI.
 func (r *ReScheduler) Execute(input *TurboActionExecutorInput) (*TurboActionExecutorOutput, error) {
 	actionItem := input.ActionItem
 	pod := input.Pod
 
 	//1. get target Pod and new hosting Node
 	node, err := r.getPodNode(actionItem)
 	if err != nil {
-		glog.Errorf("Failed to execute pod move: failed to get target pod or new hosting node: %v", err)
-		return &TurboActionExecutorOutput{}, fmt.Errorf("Failed")
+		glog.Errorf("Failed to execute pod move: %v.", err)
+		return &TurboActionExecutorOutput{}, err
 	}
 
-	//2. move pod to the node
+	//2. move pod to the node and check move status
 	npod, err := r.reSchedule(pod, node)
 	if err != nil {
-		glog.Errorf("Failed to execute pod move: %v\n %++v", err, actionItem)
+		glog.Errorf("Failed to execute pod move: %v.", err)
```

Comment:
Your point is valid that we should log the ``actionItem`` (in Error level) when action execution fails. I have updated the code to log the ``actionItem`` in ``ActionHander`` -> ``execute()``:
```
	output, err := worker.Execute(input)

	if err != nil {
		glog.Errorf("Failed to execute action %v on %s: %+v.",
			actionType, actionItem.GetTargetSE().GetEntityType(), actionItem)
		return err
	}
```

I removed the logging before because the ``actionItem`` has already been logged in the go sdk ``client_protobuf_endpoint.go`` in Info level when the action is received. The message is rather huge occupying two screens. It probably makes more sense to bump the logging to V(4) in go sdk. This can be addressed later if needed.